File: check positive or negative/CheckNumber.go
package main
import "fmt"
func main () {
	var A int
    fmt.Printf("enter the number:")
	fmt.Scanf("%d",&A)
	if A<0 {
	   fmt.Printf("the value is negative")
	}else if A>0 {
	   fmt.Printf("the value is positive")
	}else  {
	   fmt.Printf("equal")
	}
}

File: check positive or negative/CheckNumber.c
#include <stdio.h>
int main(){
    int A;
    printf("enter the number");
    scanf("%d",&A);
    if(A>0)
       printf("the value is positive");
    else if(A<0)
       printf("the value is negative");
    else if(A==0)
       printf("the value is equal to zero");
    return 0;
}